src/library_emmalloc.js

```diff
@@ -0,0 +1,24 @@
+mergeInto(LibraryManager.library, {
+  emmalloc_unclaimed_heap_memory__deps: ['emscripten_get_sbrk_ptr'],
```

Comment:
This also looks like it just used in tests.  Is it worth added this JS library just for sake of a test case?  Maybe there is some way to avoid it?

Reply:
The intent is to expose that function to public API for users of emmalloc, hence testing that it works in that test.

Reply:
But isn't that function internal-only right now?  So the fact that it has a bad name is currently not an issue?

Reply:
I agree that this should be split out into a separate PR btw.   Although I'm still not convinced about the need to shrink memory (since its not possible with wasm).